src/commands/version.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const packageConfig = require('../../package.json');
+
+/* Export as Yargs command */
+exports.command = 'version'
+exports.aliases = ['v']
+exports.desc = 'Show information about current orbit-db'
+
+exports.builder = function(yargs) {
+    return yargs
+        .usage(`Usage: $0 version`)
+}
+
+exports.handler = async(argv) => {
```

Comment:
Add a space before `(`, eg. `async (argv)`

Reply:
Same as above, optional to fix.

Reply:
Will add it :+1: